src/pages/dashboard.vue

```diff
@@ -174,10 +174,94 @@ const pendingAddress = computed(() => {
 
 const isChangeAddressDisabled = computed(() => business.currentBusiness.adminFreeze || pendingAddress.value)
 const isChangeDirectorDisabled = computed(() => business.currentBusiness.adminFreeze)
+const showChangeOfAddress = ref(false)
+
+const toggleChangeOfAddress = (show: boolean) => {
+  showChangeOfAddress.value = show
+}
+
+const goToStandaloneAddresses = () => {
+  const baseUrl = useRuntimeConfig().public.dashboardOldUrl
+  const url = `${baseUrl}/${business.currentBusinessIdentifier}/standalone-addresses?filingId=0`
+  navigateTo(url, { external: true })
+}
+
+const changeAddress = () => {
+  if (business.isEntityFirm()) {
+    const baseUrl = useRuntimeConfig().public.editApiURL
+    const url = `${baseUrl}${business.currentBusinessIdentifier}/change`
+    navigateTo(url, { external: true })
+  } else if (business.isBaseCompany()) {
+    toggleChangeOfAddress(true)
```

Comment:
e.g. here, when I read, its like toggleChangeOfAddress, param true meaning it will toggle it.